application/templates/admin/merge_data_sources.html

```diff
@@ -0,0 +1,34 @@
+{% extends "base.html" %}
+{% from "_shared/_breadcrumb.html" import breadcrumb %}
+
+{% set breadcrumbs =
+  [
+    {"url": url_for("static_site.index"), "text": "Ethnicity facts and figures"},
+    {"url": url_for('admin.index'), "text": "Admin"}
+  ]
+%}
+
+{% block pageTitle %}Merge data sources{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <h1 class="govuk-heading-xl">Merge data sources</h1>
+
+      <div class="govuk-body">All of the following data sources will be merged together.</div>
+      <form action="{{ url_for('admin.merge_data_sources_post') }}" method="POST">
+        {{ data_source_merge_form.csrf_token | default('') }}
+
+        {{ data_source_merge_form.keep }}
+
+        {% for data_source in data_sources %}
+          <input type="hidden" name="ids" value="{{ data_source.id }}" />
+        {% endfor %}
+
+        <button class="govuk-button">Merge</button>
```

Comment:
I've always tagged these up as `type="submit"` - is that not really necessary?

Reply:
Yeah, `type="submit"` is the default behaviour, so you don't need to add it. You only have to add `type="button"` if you _don't_ want it to submit. 🤷‍♂